src/libs/OptionsListUtils.js

```diff
@@ -753,7 +753,7 @@ function sortTags(tags) {
  * @param {Boolean} [isOneLine] - a flag to determine if text should be one line
```

Comment:
Please change the function description to more generic instead of specifically for category to something like
"Builds the options for the tree hierarchy via indents"

Reply:
Hi, @c3024 sorry I forgot to change this description.

Done: https://github.com/Expensify/App/pull/32337/commits/3f5b02487